chichiwang/chichiwang.dev
# Replace root route with Resume page

## Description
Linked to Issue: #3 
Create a placeholder `Resume` page, place it on the `/` route, replacing the `Home` page

## Changes
* Update `app/dataRoutes/index.ts` to replace `Home` and `HomeHandle` on the `/` route with `Resume` and `ResumeHandle`
* Create a placeholder Resume page `pages/Resume/index.tsx`
  * Create head tags for Resume page `pages/Resume/Head.tsx`
  * Create route handle for Resume page `pages/Resume/handle.tsx`

## Steps to QA
* Pull down and switch to this branch
* Run the app and ensure no issues
* Verify in browser that the `/` route renders the placeholder Resume page
